docs/usage/test/odata.md

```diff
@@ -0,0 +1,89 @@
+# Testing OData services
+
+## APIs supporting OData testing
+
+The following classes provide you a simple API so you can focus on writing your test cases without requiring deep
+knowledge of the [OData protocol 4.0](http://docs.oasis-open.org/odata/odata/v4.0/odata-v4.0-part1-protocol.html).
+
+> Hint: Current implementations follow OData protocol version 4.0
+
+| Class                                                                                                                    | Description                                                                                                                                                        |
+|--------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [io.neonbee.test.base.AbstractODataRequest](../../../src/test/java/io/neonbee/test/base/AbstractODataRequest.java)       | Base class providing functionality for creating, modifying and dispatching OData requests common to all OData request types like metadata, data or batch requests. |
```

Comment:
I'd not mention AbstractODataRequest, because you can not really use it. Especially without deep OData knowledge.

Reply:
As discussed